gulpfile.js

```diff
@@ -4,44 +4,97 @@ var path = require('path');
 var del = require('del');
 var runSequence = require('run-sequence');
 var kss = require('kss');
+var rename = require('gulp-rename');
+var browserSync = require('browser-sync').create();
 
+// Local configuration file
+var localCnf = require('./gulp-config-local.js');
```

Comment:
Do we really want to require on the local configs? What if someone wants to just run this using the default configs?

Reply:
@lauriii I made it this way to be more comfortable to use.
The local config currently can have only one element. It is the localhost url (see example.gulp-config-local.js). If someone do not want to use an external file can it is possible to write the localhost url directly to the gulpfilejs. However in that case she has to keep in mind not to commit the changes (because others can have different localhost urls.)

In the other way I put gulp-local-config.js to .gitignore so it is safe to use.

Or you mean it can be a problem if someone has no a local config file? Can that be resolved with an if statement which checks the existance of the file? If so can you help to write it correctly?

Reply:
/me is not a node-js/gulp wizard… :)

Reply:
+1 for having a local file. It's handy so everyone can have their own URL (and/or add more local items if they want).